server/pulp/server/controllers/distributor.py

```diff
@@ -237,6 +237,8 @@ def update(repo_id, dist_id, config=None, delta=None):
 
     if not valid_config:
         raise exceptions.PulpDataException(message)
+
+    distributor.last_updated = dateutils.now_utc_datetime_with_tzinfo()
```

Comment:
Rather than setting this here, can a pre-save signal be created for the Distributor model which will set this automatically. This is how it is done with ContentUnit for example: https://github.com/pulp/pulp/blob/master/server/pulp/server/db/model/__init__.py#L681-L693

Doing it with a signal has the typical object oriented design advantages. It couples behaviors with data. In this case the behavior is the maintenance of that field. By having it maintained by the design itself instead of in this controller if people don't use the controller it will still work as expected so more consistency with fewer bugs.

Not using the controllers will likely become more typical over time since so much of controller functionality deals with validation and setting of attributes both of which the mongoengine definitions can handle all on their own.